### PR TITLE
#1589 Move duplicate qualified modifier error to new format

### PR DESCRIPTION
**Pull request by lorandszakacs**
This was a rather niched error message, since most of the time it is superseded by other errors.

This corresponds to the `Parsers:1703` index from the issue description.

P.S. I now realized that I will be leaving to areas with no internet soon... so hopefully all is well with the PR, otherwise I'll have to address it in a few days 😟

### CODE REVIEW

File: compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala [hunk 1]
```diff
@@ -1694,4 +1694,12 @@ object messages {
     val explanation = ""
   }
 
+  case class DuplicatePrivateProtectedQualifier()(implicit ctx: Context)
+    extends Message(DuplicatePrivateProtectedQualifierID) {
+    val kind = "Syntax"
+    val msg = s"duplicate private/protected qualifier"
+    val explanation =
+      hl"""It is not allowed to combine `private` and `protected` modifiers even if they are qualified to different scopes""".stripMargin
```
Comment (nicolasstucki) [inline compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala]: You do not need `.stripMargin` here.

Comment (nicolasstucki) [inline compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala]: You also do not need `"""` for the string, `"` is better.

File: compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java [hunk 2]
```diff
@@ -93,7 +93,8 @@
     ModifiersNotAllowedID,
     WildcardOnTypeArgumentNotAllowedOnNewID,
     ImplicitFunctionTypeNeedsNonEmptyParameterListID,
-    WrongNumberOfParametersID
+    WrongNumberOfParametersID,
+    DuplicatePrivateProtectedQualifierID
```
Comment (nicolasstucki) [inline compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java]: Add a `,` after `DuplicatePrivateProtectedQualifierID`

**Comment by lorandszakacs** [comment]
@nicolasstucki, I addressed the issues. Thanks!

**Comment by nicolasstucki** [comment]
@lorandszakacs thanks a lot!

**Pull request merged**